Gemfile

```diff
@@ -4,10 +4,10 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-gem "decidim", "0.21.0"
-gem "decidim-consultations", "0.21.0"
+gem "decidim", "0.22.0"
```

Comment:
Defining a `DECIDIM_VERSION = "0.22.0"` above this line so we can reuse it? Look what [Decidim BCN](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/blob/4bbc65d99a0a3820d8d6d69e58af0df71a382247/Gemfile) and [Metadecidim](https://github.com/decidim/metadecidim/blob/f8bcbfaa2aefd0dd51daf5dd2a93c8198265c118/Gemfile#L7) did.